packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -432,7 +432,9 @@ export class SchedulerModel {
         const logs = await this.database(SchedulerLogTableName)
 
             .select()
-            .whereIn(`scheduler_uuid`, uniqueSchedulerUuids);
+            .whereIn(`scheduler_uuid`, uniqueSchedulerUuids)
+            .limit(1500);
```

Comment:
This is still a huge to be rendered on one page, what about 100?

Reply:
I don't think many teams have schedules that run every minute 🤔 - that would definitely be an outlier ! It's more likely to have something that runs once or twice daily or once weekly

Reply:
Who has a scheduled delivery every minute? I think that's an edge case.

Also I've just realised there is no order by clause here, so will you get all the most recent results?

Instead why don't we:
1. Index on uuid + created_at
2. filter on uuid + created_at in the last 2 days?
3. order by created_at desc
4. limit 1500

Most people won't have more than a few deliveries in the last 2 days, so they won't be forced to render a huge table of results. But people that really do have 1 a minute will get a day of results.

Reply:
Yeah, we can do that, but I'd increase it to 3-4 days, so you cover the weekend

Reply:
I prefer to just limit the last 100 recent results than filtering by date if you think it is an edge case

Reply:
Ok let's do that! Order by created at date and 100 results max